Source/UIControl+LambdaKit.swift

```diff
@@ -99,8 +99,19 @@ extension UIControl {
             self.events?[event] = nil
             for wrapper in wrappers {
                 self.removeTarget(wrapper, action: #selector(ControlWrapper.invoke(_:)),
-                    for: UIControlEvents(rawValue: event))
+                                  for: UIControlEvents(rawValue: event))
             }
         }
     }
+
+    /// Convenience function for replacing a control's handlers for a given event with a new handler.
+    ///
+    /// - parameter controlEvents: A bitmask specifying the control events that the handler will replace.
+    /// - parameter handler:       A closure representing an action message, with an argument for the sender.
+    public func replaceEventHandler(forControlEvents controlEvents: UIControlEvents,
```

Comment:
`replace...` vs `set...`? Feels weird to call `replace...` the first time around. To me it's pretty clear that `set` means to overwrite what's there and `add` appends .. that's already a convention in UIKit.